frontend/src/metabase/components/ListSearchField/ListSearchField.tsx

```diff
@@ -24,6 +24,7 @@ export default function ListSearchField({
 
   return (
     <TextInput
+      data-testid="list-search-field"
```

Comment:
This prop is duplicated, we already have `data-testid="list-search-field"` below

Reply:
ah this is a merge issue